File: Leetcode/Algorithms/Easy/Find Numbers with Even Number of Digits/Find Numbers with Even Number of Digits_2.go
func findNumbers(nums []int) int {
	count := 0

	for _, v := range nums {
		if (v >= 10 && v <= 99) || (v >= 1000 && v <= 9999) || v == 100000 {
			count++
		}
	}

	return count
}

File: Leetcode/Algorithms/Easy/Find Numbers with Even Number of Digits/Find Numbers with Even Number of Digits_2.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        count = 0

        for v in nums:
            if (v >= 10 and v <= 99) or (v >= 1000 and v <= 9999) or v == 100000:
                count += 1

        return count
        